module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -1203,30 +1203,40 @@ public function checkedoutAction()
         // By default, assume we will not need to display a renewal form:
         $renewForm = false;
 
+        // Get paging setup:
+        $config = $this->getConfig();
+        $pageOptions = $this->getPaginationHelper()->getOptions(
+            (int)$this->params()->fromQuery('page', 1),
+            $this->params()->fromQuery('sort'),
+            $config->Catalog->checked_out_page_size ?? 50,
+            $catalog->checkFunction('getMyTransactions', $patron)
+        );
+
         // Get checked out item details:
-        $result = $catalog->getMyTransactions($patron);
+        $result = $catalog->getMyTransactions($patron, $pageOptions['ilsParams']);
 
-        // Get page size:
-        $config = $this->getConfig();
-        $limit = isset($config->Catalog->checked_out_page_size)
-            ? $config->Catalog->checked_out_page_size : 50;
+        // Support also older driver return value:
+        if (!isset($result['count'])) {
```

Comment:
Would it be more seamless to put this normalization into \VuFind\ILS\Connection? I'm thinking ahead to #966, which introduces a call to getMyTransactions in a second place... centralizing the formatting might make us less likely to end up with future inconsistencies.